src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -1,121 +1,202 @@
-        font-weight: 600;
-      `}
-    >
-      Apps in development
-    </h1>
-    <AppsGrid>
-      {appsInDevelopment.map((app, i) => (
-        <Main key={i}>
-          <Icon>
-            <AppIcon size={64} src={app.icon} />
-          </Icon>
-          <Name>{app.name}</Name>
-          <TagWrapper>
-            <Tag mode="new">{app.status}</Tag>
-          </TagWrapper>
-          <Description color={theme.textSecondary}>
-            {app.description}
-          </Description>
-          <Action href={app.link} target="_blank">
-            <Text weight="bold" color={theme.textSecondary}>
-              Learn more
-            </Text>
-          </Action>
-        </Main>
-      ))}
-    </AppsGrid>
-  </div>
-))
+  return (
+    <React.Fragment>
+      <Info
+        css={`
+          margin: 0 ${compactMode ? 2 * GU : 0}px;
+          margin-bottom: ${2 * GU}px;
+        `}
+      >
+        You will soon be able to install new apps into your Aragon organization
+        from here. In the meantime, you can use our{' '}
+        <Link
+          href="https://hack.aragon.org/docs/cli-dao-commands#dao-install"
+          target="_blank"
+        >
+          CLI guide
+        </Link>{' '}
+        and learn{' '}
+        <Link href="" target="_blank">
+          how to create apps yourself.
+        </Link>
+        You can also preview some of the apps being developed.
+      </Info>
+      <CardLayout columnWidthMin={columnWidthMin} rowHeight={rowHeight}>
+        {appsInDevelopment.map((app, i) => (
+          <AppCard key={i} app={app} compactMode={compactMode} />
+        ))}
+      </CardLayout>
+    </React.Fragment>
+  )
+})
 
-const AppsGrid = styled.div`
-  display: grid;
-  grid-auto-flow: row;
-  grid-gap: 25px;
-  justify-items: start;
-  grid-template-columns: 1fr;
-  ${breakpoint(
-    'medium',
-    `
-      grid-template-columns: repeat(auto-fill, 224px);
-    `
-  )};
-`
+function AppCard({ app, compactMode, onOpen, ...props }) {
+  const theme = useTheme()
+  const { link, icon, name, status, description } = app
+  const handleClick = useCallback(() => {
+    if (link) {
+      window.open(link, '_blank', 'noopener')
+    }
+  }, [link])
+
+  return (
+    <Card onClick={handleClick}>
+      <Main compactMode={compactMode}>
+        <StyledIconExternal
+          compactMode={compactMode}
+          theme={theme}
+          link={link}
+        />
+        <Icon compactMode={compactMode}>
+          <AppIcon size={9 * GU} src={icon} radius={12} />
+        </Icon>
+        <Name compactMode={compactMode}>{name}</Name>
+        <TagWrapper compactMode={compactMode} link={link}>
+          <Tag mode="indicator">{status}</Tag>
+        </TagWrapper>
+        <Description theme={theme} compactMode={compactMode}>
+          {description}
+        </Description>
+      </Main>
+    </Card>
+  )
+}
 
-const Main = styled(Card).attrs({ width: '100%', height: '288px' })`
+const Main = styled.section`
```

Comment:
Maybe we can name this `CardMain` or etc. to not confuse it with the main component of `DiscoverApps`